app/src/main/java/org/fossasia/openevent/app/events/EventDetailActivity.java

```diff
@@ -0,0 +1,90 @@
+package org.fossasia.openevent.app.events;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.data.models.Event;
+import org.fossasia.openevent.app.event.EventContainerFragment;
+import org.fossasia.openevent.app.listeners.OnFragmentInteractionListener;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * An activity representing a single Event detail screen. This
+ * activity is only used narrow width devices. On tablet-size devices,
+ * item details are presented side-by-side with a list of items
+ * in a {@link EventListActivity}.
+ */
+public class EventDetailActivity extends AppCompatActivity implements OnFragmentInteractionListener {
+
+    @BindView(R.id.detail_toolbar)
+    Toolbar toolbar;
+
+    private Event initialEvent;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_event_detail);
+
+        ButterKnife.bind(this);
+
+        setSupportActionBar(toolbar);
+
+        // Show the Up button in the action bar.
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        initialEvent = getIntent().getParcelableExtra(EventListActivity.EVENT_KEY);
+
+        setTitle(initialEvent.getName());
+
+        // savedInstanceState is non-null when there is fragment state
```

Comment:
It's a standard Android knowledge, it isn't helpful to the project per se. If it were for beginners of Android, it would have been good. Our comments should ideally be not documenting(re) the standard Android behavior